api/app/pagination.py

```diff
@@ -29,33 +76,37 @@ def paginate_queryset(self, dynamo_queryset, request, view=None):
             for identity_document in dynamo_queryset["Items"]
         ]
 
-    def get_next_link(self) -> str:
-        url = self.request.build_absolute_uri()
-        next_url = (
-            replace_query_param(url, "last_evaluated_key", self.last_evaluated_key)
-            if hasattr(self, "last_evaluated_key")
-            else None
-        )
-        return next_url
-
-    def get_previous_link(self) -> str:
-        url = self.request.build_absolute_uri()
-        previous_url = (
-            replace_query_param(
-                url, "last_evaluated_key", self.previous_last_evaluated_key
-            )
-            if self.previous_last_evaluated_key
-            else None
-        )
-        return previous_url
-
     def get_paginated_response(self, data) -> Response:
+        """
+        Note: "If the size of the Query result set is larger than 1 MB, ScannedCount
+                and Count represent only a partial count of the total items"
+        ref: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.html#Query.Count
+
+        Tl;dr: there is no straightforward way to get the total number of items from dynamodb, that's why we don't
+                include count in the response
+        """
         return Response(
             OrderedDict(
                 [
                     ("results", data),
-                    ("previous", self.get_previous_link()),
-                    ("next", self.get_next_link()),
+                    (
+                        "last_evaluated_key",
+                        self.last_evaluated_key
+                        if hasattr(self, "last_evaluated_key")
+                        else None,
+                    ),
                 ]
             )
         )
+
+    def get_paginated_response_schema(self, schema):
```

Comment:
Seems odd that we need this as well as the `get_paginated_response` method on the Inspector class above?

Reply:
Yeah, it's not useful anymore